actix-http/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "actix-http"
-version = "1.0.1"
+version = "2.0.0-alpha"
```

Comment:
Could you not bump up version here? It may cause conflict when merging (into) master.

Reply:
@dunnock

Reply:
Fixed. I also temporarily excluded local actix-http resolution from actix-web so that it compiles, next step would be to adopt changes to actix-web and awc

Reply:
Merged in last updates from master, fixed tests to account for actix-http is not compatible with other workspace packages now